prompts/github.go

```diff
@@ -556,18 +553,17 @@ func defaultPublishingFile() *config.PublishWorkflow {
 		},
 		On: config.PublishOn{
 			Push: config.Push{
-				Paths: []string{
-					"RELEASES.md",
-					"*/RELEASES.md",
-				},
+				Paths: []string{},
```

Comment:
This doesn't seem right

Reply:
We fill it in during configurePublishing. Same thing we are doing with Secrets. It's just no longer a hardcoded value